controls/entry/styling.md

```diff
@@ -15,25 +15,34 @@ The Entry provides styling options for setting the appearance of its border and
 
 Style the entry using the following properties:
```

Comment:
- Capitalize the first letter of the component name in all occurrences.
- Remove the `## Entry Style` heading above as it duplicates the title and doesn't add value.